docker/utils/setup_environment.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+# source robot environment
+source /run/husarion/robot_config.env
+
+# check if /config directory is empty except it may contain common directory inside
+if [ ! "$(ls -A /config | grep -v common)" ]; then
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)